### PR TITLE
Replaces the Kudzu flytrap pod effects sprite

### PR DESCRIPTION
**Pull request by ATP-Engineer**
## What Does This PR Do

Replaces an old kudzu pod sprite

## Why It's Good For The Game

Pod sprite is VERY OLD. This one makes the rare kudzu wars you see more easy to look at.
image

## Images of changes

OLD
image

NEW
image

Side by side
image

## Testing

scaled to reference, ensuring that the space vine effect didnt break/not connect

## Changelog
:cl:
imageadd: Replaces Space vine man-eater pod sprite
/:cl:

### CODE REVIEW

**Review by S34NW — APPROVED** [review]
Technically sound, though @McRamon might have something to say before full approval

**Review by Sirryan2002 — APPROVED** [review]

**Pull request merged**
